### PR TITLE
Hero link to filtered matches by hero_id

### PR DESCRIPTION
**Pull request by blukai**

### CODE REVIEW

**Comment by howardchung** [comment]
For peers this is done via a link on the number of matches.  They should be consistent?

**Comment by blukai** [comment]
`transformations.hero_id` used in different places. So I just made a thing like for subtitle that determine where `heroName` should point to matches played on this hero and where not.

**Comment by howardchung** [comment]
Yeah but we could put the link on the MP column and leave transformations.hero_id as is?

**Comment by howardchung** [comment]
Try merging master into this, should fix the CI

File: src/components/Player/Pages/Heroes/playerHeroesColumns.jsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import { TablePercent } from 'components/Visualizations';
 import strings from 'lang';
 
-export const playerHeroesOverviewColumns = [{
+export const playerHeroesOverviewColumns = playerId => [{
   displayName: strings.th_hero_id,
   tooltip: strings.tooltip_hero_id,
   field: 'hero_id',
-  displayFn: transformations.hero_id,
+  displayFn: row => <Link to={`/players/${playerId}/matches?hero_id=${row.hero_id}`}>{transformations.hero_id(row)}</Link>,
```
Comment (howardchung) [inline src/components/Player/Pages/Heroes/playerHeroesColumns.jsx]: games column, not hero_id column?

Comment (blukai) [inline src/components/Player/Pages/Heroes/playerHeroesColumns.jsx]: what❔

Comment (howardchung) [inline src/components/Player/Pages/Heroes/playerHeroesColumns.jsx]: the filter link for peers/pros is on the column with the number of games.  For heroes it should be the same (so the column with field: 'games'?

Comment (blukai) [inline src/components/Player/Pages/Heroes/playerHeroesColumns.jsx]: No, I don't think so. There is no reason to add new column when we can make hero clickable.

Comment (howardchung) [inline src/components/Player/Pages/Heroes/playerHeroesColumns.jsx]: It's not a new column, it's the "matches played" column that already exists.

**Comment by howardchung** [comment]
LGTM.  perhaps in the future we can also add the links for with/against heroes, but i guess we can do that if someone asks for it :)

**Pull request merged**